<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QPushButton" name="OKButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Basic PE Header Information</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>EntryPoint</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="Subsystem"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>ImageBase</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="ImageBase"/>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>NumberOfsections</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="NumberOfsections"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="EntryPoint"/>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Subsystem</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="Details">
        <property name="text">
         <string>Details</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="BaseOfCode"/>
      </item>
      <item row="3" column="5">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>FileAlignment</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>NumOfRvaAndSizes</string>
        </property>
       </widget>
      </item>
      <item row="7" column="4">
       <widget class="QLineEdit" name="NumOfRvaAndSizes"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>SizeOfImage</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>路径</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="4">
       <widget class="QLineEdit" name="filelocation"/>
      </item>
      <item row="8" column="6">
       <widget class="QPushButton" name="openfile">
        <property name="text">
         <string>打开</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>SizeOfHeaders</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="SectionAlignment"/>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>BaseOfData</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>BaseOfCode</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>TimeDateStamp</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLineEdit" name="SizeOfHeaders"/>
      </item>
      <item row="1" column="6">
       <widget class="QPushButton" name="directories">
        <property name="text">
         <string>Directories</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>SectionAlignment</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QLineEdit" name="Characteristics"/>
      </item>
      <item row="5" column="4">
       <widget class="QLineEdit" name="Checksum"/>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="FileAlignment"/>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="BaseOfData"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="SizeOfImage"/>
      </item>
      <item row="3" column="6">
       <widget class="QPushButton" name="File_Location_Calculator">
        <property name="text">
         <string>File Location Calculator</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Checksum</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="TimeDateStamp"/>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Characteristics</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Magic</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLineEdit" name="Magic"/>
      </item>
      <item row="2" column="6">
       <widget class="QPushButton" name="Import">
        <property name="text">
         <string>Sections Table</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>SizeOfOptionlHeader</string>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QLineEdit" name="SizeOfOptionlHeader"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>OKButton</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>758</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>806</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
